lib/ansiblelint/utils.py

```diff
@@ -382,7 +403,8 @@ def normalize_task_v2(task):
             task_pprint = pp.pformat(task)
         except ImportError:
             task_pprint = task
-        raise SystemExit("Couldn't parse task at %s (%s)\n%s" % (task_info, e.message, task_pprint))
+        logger.fatal("Couldn't parse task at %s (%s)\n%s" % (task_info, e.message, task_pprint))
```

Comment:
There's no `fatal` here: https://docs.python.org/3/library/logging.html. Did you mean
```suggestion
        logger.critical("Couldn't parse task at %s (%s)\n%s", task_info, e.message, task_pprint)
```